gmso/core/angle.py

```diff
@@ -99,3 +131,14 @@ def __setattr__(self, key, value):
             super(Angle, self).__setattr__("angle_type", value)
         else:
             super(Angle, self).__setattr__(key, value)
+
+    @model_validator(mode="before")
+    @classmethod
+    def set_dependent_value_default(cls, data):
```

Comment:
I think a concise doc string for this method would be helpful.

Reply:
Done.